docs/source/release_notes/release-notes.rst

```diff
@@ -43,6 +43,22 @@ develop
 
 .. replace this with the release date and the above with v<tag> (the v makes the permalinks nice)
 
+.. list-table::
+    :widths: 5 40
+    :header-rows: 1
+
+    *    - Type
+         - Change
+    
+    *    -
+         -
+
+=======
+v0.79.0
+=======
+
+20 March 2018
+
 .. list-table::
```

Comment:
The Guava dev break should probably go at the top (we normally put breaking changes on top).